drills/drills.py

```diff
@@ -22,11 +25,13 @@ class Prompt:
     def __init__(self,
                  slug: str,
                  messages: List[str],
+                 should_store_response: Optional[bool] = False,
```

Comment:
@pdsarin this field exists on the `PromptSchema`, but didn't exist on the `Prompt`. Do we need it?

Reply:
oops. Actually we should delete it from the schema. No longer needed. Mind fitting that in here?

Reply:
Deleted.